README.md

```diff
@@ -1,14 +1,146 @@
+```dart
+class HotReload extends StatelessWidget {
+  const HotReload({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Widgetbook(...);
+  }
+}
+```
+
+### Running the Widgetbook
+
+To run the `Widgetbook` create a new main method, e.g. in `stories/main.dart`:
+
+```dart
+void main() {
+  runApp(HotReload());
+}
+```
+
+Run the `Widgetbook` main method by executing `flutter run -t stories/main.dart`.
+
+### Inject your widgets
+
+Your widgets can be organized into different areas of interest by using `Category`, `Folder`, `WidgetElement` and `Story`.
+
+```dart
+class HotReload extends StatelessWidget {
+  const HotReload({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Widgetbook(
+      categories: [
+        Category(
+          name: 'widgets',
+          widgets: [
+            WidgetElement(
+              name: '$CustomWidget',
+              stories: [
+                Story(
+                  name: 'Default',
+                  builder: (context) => CustomWidget(),
+                ),
+              ],
+            ),
+          ],
+          folders: [
+            Folder(
+              name: 'Texts',
+              widgets: [
+                WidgetElement(
+                  name: 'Normal Text',
+                  stories: [
+                    Story(
+                      name: 'Default',
+                      builder: (context) => Text(
+                        'The brown fox ...',
+                      ),
+                    ),
+                  ],
+                ),
+              ],
+            ),
+          ],
+        ),
+      ],
+      appInfo: AppInfo(
+        name: 'Widgetbook Example',
+      ),
+    );
+  }
+}
+```
+
+## Set storybook name
+
+Customize `Widgetbook`'s name regarding to the project by using appInfo:
+
+```dart
+Widgetbook(
+  appInfo: AppInfo(
+    name: 'Your apps name',
+  ),
+)
+```
+
+## Adding Themes
+
+Import your app's theme for a realistic preview by using `Widgetbook`'s `lightTheme` and `darkTheme` properties:
+```dart
+Widgetbook(
+  lightTheme: lightTheme,
+  darkTheme: darkTheme,
+)
+```
+
+## Add devices
+
+Customize the preview by defining preview devices: 
+
+```dart
+Widgetbook(
+  devices: [
+    Apple.iPhone11,
+    Apple.iPhone12,
+    Samsung.s10,
+    Samsung.s21ultra,
+  ]
+)
+```
+
+Right now there is a predefinied short list of devices but let us know what you need in your [Discord](https://discord.gg/zT4AMStAJA). We will expand the list of predefinied devices in the future!
+
+# Upcoming features
+
+◼ Search bar
+
+◼ More predefined devices
 
-For help getting started with Flutter, view our 
-[online documentation](https://flutter.dev/docs), which offers tutorials, 
-samples, guidance on mobile development, and a full API reference.
+As already said: Let us know if you need anything! 
```

Comment:
As already said: We love hearing from users! Link zu discord und Link zu Calendly

Reply:
hab das einfach geloescht